db/schema.rb

```diff
@@ -1254,11 +1266,10 @@
   add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
   add_foreign_key "active_storage_variant_records", "active_storage_blobs", column: "blob_id"
   add_foreign_key "oauth_openid_requests", "oauth_access_grants", column: "access_grant_id", on_delete: :cascade
+  add_foreign_key "payment_intents", "users", column: "initiated_by_id"
   add_foreign_key "paypal_records", "paypal_records", column: "parent_record_id"
   add_foreign_key "sanity_check_exclusions", "sanity_checks"
   add_foreign_key "sanity_checks", "sanity_check_categories"
-  add_foreign_key "stripe_payment_intents", "stripe_records"
-  add_foreign_key "stripe_payment_intents", "users"
```

Comment:
At least this one foreign key should be maintained

Reply:
I believe it is maintained by line 1275 - note the change in name from `user_id` to `iniatiated_by_id`

Reply:
Ah, gotcha.